### ISSUE TITLE
Missing part in pokemon name #866

### ISSUE THREAD

**Issue by mrgiel** [PokeAPI/pokeapi#537]
Pokemon #866 is missing a part in its name. In the API the pokemon name is Mr, but it should be Mr. Rime (or mr-rime since mr-mime is also with a -)

**Comment by Naramsim** [comment]
Hello, yes, we are aware of it. I already pushed a fix in our data repository. I'm waiting on another PR to be merged and then release.

**Comment by cgbsu** [comment]
I think this is solved?

**Comment by Naramsim** [comment]
Yep! Thanks!!
